index.html

```diff
@@ -0,0 +1,181 @@
+          <li class="last"><a href="#">Log out</a></li>
+        </ul>
+      </div>
+
+      <nav>
+        <!-- Desktop Navbar -->
+        <div class="desktop-nav">
+          <img class="page-logo" src="./Images/logo.png" alt="page logo">
+          <ul class="desktop-nav-list">
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#services">Our Services</a></li>
+            <li><a href="#team-container">Our Team</a></li>
+            <li><a href="#supporters">Sponsors</a></li>
+            <li class="last"><a class="list-btn" href="#">Our Projects</a></li>
+          </ul>
+        </div>
+
+        <!-- Mobile Navbar -->
+        <div>
+          <img id="hamburger-icon" class="union-icon" src="./Images/hamburger.png" alt="hamburger icon">
+
+          <!-- close icon-->
+          <a href="#" class="close-icon hidden">
+            <img src="./Images/close.png" alt="close icon">
+          </a>
+
+          <!-- mobile menu -->
+          <ul class="mobile-menu hidden">
+            <li class="mobile-list"><a href="./about.html">About</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#services">Our Services</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#team-container">Our Team</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#supporters">Sponsors</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#">Our Projects</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="headline">
+      <article class="healine-title">
+        <p class="to-viewers">"Hello, front-end lovers!"</p>
+        <h1 class="frontend">Front-End School, best place to learn frontend technologies!</h1>
+      </article>
+
+      <article class="intro">
+
+        <p class="short-info">
+          Here we provide you with a structured pathway that will teach you all you need to know to become a front-end
+          web developer. Work through each section, learning new skills (or improving existing ones) as you go along.
+        </p>
+
+        <div class="my-info">
+          <p class="date">Monday to Friday ~ Full Time Program</p>
+          <p class="address">@Chandigarh, Punjab(India), Front-End School</p>
+        </div>
+
+      </article>
+    </section>
+
+  </div>
+  <!-- Headline and Navbar End -->
+
+  <!-- Main section (Cards) start -->
+  <main id="services" class="our-services">
+    <section class="main">
+      <h2 class="service-heading">Our Services</h2>
+      <div class="line"></div>
+
+      <section class="my-cards">
+
+        <article class="card">
+          <img class="card-img" src="./Images/one.png" alt="First card image">
+          <h3 class="card-title">Web Development</h3>
+          <p class="card-content">We will help you to learn the latest top frontend technologies. Such as: HTML5, CSS3,
+            BootStrap5, JS, React
+            and more. </p>
+        </article>
```

Comment:
I love the design of your card, but the card icons are white in the original design, in your case it's white. Additionaly don't you think you need to have some space between the `h3` and the `p` tags, they're both so close to each other somehow. Kindly make sure to update it.
![image](https://user-images.githubusercontent.com/34779928/205143452-6942ab7b-2fe3-4f5f-8183-ac41209cfaa4.png)